tests-legacy/Integration/PrestaShopBundle/Routing/Converter/LegacyUrlConverterTest.php

```diff
@@ -152,13 +152,6 @@ public static function getMigratedControllers()
             'admin_emails_delete_all' => ['/configure/advanced/emails/delete-all', 'AdminEmails', 'deleteAll'],
             'admin_emails_delete' => ['/configure/advanced/emails/delete/42', 'AdminEmails', 'deletemail', ['id_mail' => 42]],
 
-            'admin_metas_index' => ['/configure/shop/seo-urls/', 'AdminMeta'],
-            'admin_metas_search' => ['/configure/shop/seo-urls/', 'AdminMeta', 'search'],
-            'admin_metas_delete' => ['/configure/shop/seo-urls/42/delete', 'AdminMeta', 'deletemeta', ['id_meta' => 42]],
-            'admin_metas_delete_bulk' => ['/configure/shop/seo-urls/delete-bulk', 'AdminMeta', 'submitBulkdeletmeta'],
-            'admin_metas_save_options' => ['/configure/shop/seo-urls/options', 'AdminMeta', 'update'],
-            'admin_metas_generate_robots_file' => ['/configure/shop/seo-urls/generate-robots-file', 'AdminMeta', 'submitRobots'],
-
             //This url is not ready to be migrated yet, the SF controller redirects to the legacy url
             // 'admin_meta_list_create' => ['/configure/shop/seo-urls/create', 'AdminMeta', 'addmeta'],
```

Comment:
What about those 😄 ?

Reply:
its part of the legacy controller `AdminMeta` which I have erased - it was used to test legacy url converting only and since I have no legacy code left in this PR then it must be removed :D

Reply:
Yes but the lines after ? Lines 162 and more

Reply:
removed the comments - url converted for whole `MetaController` is not needed due to it does not have any legacy `AdminMetaController` association.